packages/core/test/helpers/constants.ts

```diff
@@ -33,6 +34,10 @@ export const predefinedNodesTypes: INodeTypeData = {
 		type: new Start(),
 		sourcePath: '',
 	},
+	'n8n-nodes-base.manualTrigger': {
+		type: new ManualTrigger(),
+		sourcePath: '',
+	},
```

Comment:
I needed an actual trigger to test the `runPartialWorkflow2`, especially the call to `findTriggerForPartialExecution`.

The start node above is not marked as a trigger, so I added the manual trigger here.